### PR TITLE
Fix undo meta tags

### PR DESCRIPTION
**Pull request by dreikanter**
This patch updates meta for the `undo` command, following the [reference implementation](https://github.com/logux/server/blob/master/base-server.js#L676-L682).

References:

- Original issue: https://github.com/logux/logux_rails/issues/73
- [Reference JS implementation](https://github.com/logux/server/blob/master/base-server.js#L676-L682)
- [Reference spec](https://github.com/logux/server/blob/master/test/base-server.test.js#L615)
- [Logux actions spec](https://github.com/logux/logux/blob/f643d847a8d1b2c439b292c15a1e6d2bbb7332a3/3-concepts/2-action.md)

### CODE REVIEW

**Comment by ai** [comment]
Great that you are planning to fix this issue :+1:

File: lib/logux.rb [hunk 1]
```diff
@@ -78,7 +78,7 @@ def self.add_batch(commands)
   def self.undo(meta, reason: nil, data: {})
     add(
       data.merge(type: 'logux/undo', id: meta.id, reason: reason),
-      Logux::Meta.new(clients: [meta.client_id])
+      meta.undo_meta
```
Comment (ai) [inline lib/logux.rb]: I have an idea to include `action` to `logux/undo`at some moment in the future. But I think we can do it in this way right now and refactor it later.

File: lib/logux/meta.rb [hunk 2]
```diff
@@ -32,5 +32,31 @@ def time
...
+    def undo_meta
+      self.class.new(undo_meta_hash)
+    end
+
+    private
+
+    RESEND_KEYS = %w[
+      users
+      nodes
+      reasons
```
Comment (ai) [inline lib/logux/meta.rb]: `reasons` is not part of resend keys. Every Logux node has own reasons (server reasons != client reasons).

Comment (ai) [reply lib/logux/meta.rb]: But I agree that `undo` action should have the same `reasons` as origin action (they must be both in the log). Maybe we can copy `reasons` from undid action in a different way?

File: lib/logux.rb (hunk 1 above)
Comment (dreikanter) [reply lib/logux.rb]: Thank you for the review! I'll finish the edits here after current logux_rails/logux-rack updates are integrated.

**Comment by dreikanter** [comment]
After recent refactoring `Logux.undo` and other core features were moved to Rails-agnostic `logux-rack` gem. So this patch belongs to a different project now :) I will close this PR and push the changes to a new branch. Reference to a new PR will be in the next comment.

**Comment by dreikanter** [comment]
Moved: https://github.com/logux/logux-rack/pull/6

**Pull request closed without merging**